README.md

```diff
@@ -1,85 +1,155 @@
-| `/question/edit`               | POST   | Used to edit question                               | `id` <br> `title` <br> `complexity` <br> `description` <br> `tags`    | Yes                                | Yes                              |
-| `/question/getAll`             | GET    | Used to get all the questions from the database     | -                                                                     | Yes                                | No                               |
-| `/question/getQuestionDetails` | GET    | Used to get the details of the specified question   | `id`                                                                  | Yes                                | No                               |
-| `/user/change-password`        | POST   | Used to change user password                        | `id` <br> `currentPassword` <br> `newPassword` <br> `confirmPassword` | Yes                                | No                               |
-| `/user/delete`                 | POST   | Used to delete user                                 | `id`                                                                  | Yes                                | No                               |
-| `/user/login`                  | POST   | Used to login                                       | `email` <br> `password`                                               | No                                 | -                                |
-| `/user/read`                   | POST   | Used to get user information                        | `id` or `email`                                                       | Yes                                | No                               |
-| `/user/readAll`                | GET    | Used to get all users information                   | -                                                                     | Yes                                | Yes                              |
-| `/user/signup`                 | POST   | Used to create new user                             | `email` <br> `password` <br> `confirmPassword`                        | No                                 | -                                |
-| `/user/update`                 | POST   | Used to update user information (username)          | `id` <br> `username`                                                  | Yes                                | No                               |
-
-- `auth` API (port 5001) contains all the authorization related endpoints.
-- `question` API (port 3001) contains all the question data related endpoints.
-- `user` API (port 4001) contains all the user data related endpoints (including authentication).
-- Note that if the API path requires JWT token to be in the header, it means the user has to be logged in.
+## Testing
+
+### Dockered Microservices
+
+At the root directory, open a terminal 
+to create and start containers of the micro-services.
+
+```shell
+docker compose up -d
```

Comment:
Should we include mysql should not be running before running this command?

Reply:
I shall mention this under Requirements~Software.

As the way to stop mysql varies for different OS. I am hesitant to put it near where the commands are, since it might give the expectation for the command to be provided too.